.gitattributes

```diff
@@ -0,0 +1,4 @@
+*.sh text eol=lf
+gradlew text eol=lf
+blackbox/bin/** text eol=lf
+app/src/main/dist/bin/crate eol=lf
```

Comment:
The documentation for this says:

> When using `text eol=lf`, Git will always convert line endings to `LF` on checkout. You should use this for files that must keep `LF` endings, even on Windows. [1]

So, I believe this is the right decision. Thank you.

[1] https://docs.github.com/en/github/getting-started-with-github/configuring-git-to-handle-line-endings#per-repository-settings

Reply:
Hi Andreas, thanks for reviewing! 
I'm using LF so that CR doesn't prevent running scripts which are meant to be run in WSL.
Regarding gradlew - it's even though it's supposed to be used not on WSL but on Windows as well it works without CR - and we need LF here to be able to run `gradlew itest'  from WSL.